CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
   - web_viewer: `2023.1.16388-1`
 
 **BUG FIXES**
+- Fix issue making job fail when submitted as active directory user from login nodes.
```

Comment:
Nitpick: should we expand on this with a sub-bullet, saying that this was due to an incomplete configuration of AD on the head node?

Reply:
Makes sense. Fixed in next rev.